tasks/install.yml

```diff
@@ -30,5 +35,6 @@
     - { name: "gc"}
     - { name: "master"}
     - { name: "monitor", args: "--address 0.0.0.0" }
+    - { name: "proxy",   args: "-p {{ accumulo_home }}/proxy/proxy.properties" }
```

Comment:
can we install only the services that are really needed on the slave machines? using 'when' condition